### PR TITLE
New RP version for EventHub 

### PR DESCRIPTION
**Pull request by v-Ajnava**
regenerating SDK for the new swagger for RP version 2017-04-01 
New RP version swagger: 
https://github.com/Azure/azure-rest-api-specs/pull/1305
https://github.com/Azure/azure-rest-api-specs/pull/1357
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj [hunk 1]
```diff
@@ -9,5 +9,6 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>2.0.0</Version>
```
Comment (shahabhijeet) [inline src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj]: @v-Ajnava let's stick with <VersionPrefix> for now.

**Review by shahabhijeet — APPROVED** [review]

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
let's stick with versionPrefix> for now.

File: src/SDKs/EventHub/Management.EventHub/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides developers with a library to create and manage all Azure Event Hubs resources.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/EventHub/Management.EventHub/Properties/AssemblyInfo.cs]: @v-Ajnava also AssemblyVersion needs to be bumped

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/EventHub/Management.EventHub/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (v-Ajnava) [inline src/SDKs/EventHub/Management.EventHub/Properties/AssemblyInfo.cs]: Done

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**